content/home/examples/an-application.md

```diff
@@ -1,7 +1,7 @@
 ---
-title: An Application
+title: Una aplicación
 order: 2
 domid: todos-example
 ---
 
-Using `props` and `state`, we can put together a small Todo application. This example uses `state` to track the current list of items as well as the text that the user has entered. Although event handlers appear to be rendered inline, they will be collected and implemented using event delegation.
+Usando `props` y `state`, podemos integrar todo en una pequeña aplicación de tareas pendientes. Este ejemplo usa `state` para tener un control de la lista actual de objetos así como el texto que el usuario ha ingresado. A pesar que los `event handlers` parecen ser renderizados inline, estos serán recolectados e implementados usando una delegación de eventos.
```

Comment:
The original text says "using event delegation". If we were to translate what you wrote, it would say "using an event delegation", which is different.

I suggest removing `una`.